maven/plugins/common/src/main/java/com/vmware/pscoe/maven/plugins/AbstractInstallNodeDepsMojo.java

```diff
@@ -80,10 +83,41 @@ public void execute() throws MojoExecutionException, MojoFailureException {
 				.execute(getLog());
         }
 
-        new ProcessExecutor()
-			.name("Dependency installation")
-			.directory(project.getBasedir())
-			.command(deps)
-			.execute(getLog());
+        getLog().info("deps length  " + deps.stream().mapToInt(String:: length).sum());
+
+		for(String path : deps)
+		{
```

Comment:
Please format the code by using the configuration stored in .editorconfig file.

Reply:
Code formatted